platforms/quilt/src/main/java/com/dfsek/terra/quilt/QuiltPlatform.java

```diff
@@ -17,43 +17,26 @@
 
 package com.dfsek.terra.quilt;
 
-import ca.solostudios.strata.Versions;
-import ca.solostudios.strata.parser.tokenizer.ParseException;
-import ca.solostudios.strata.version.Version;
 import org.jetbrains.annotations.NotNull;
 import org.quiltmc.loader.api.QuiltLoader;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 import java.io.File;
 import java.util.Collection;
 import java.util.stream.Collectors;
-import java.util.stream.Stream;
 
-import com.dfsek.terra.addon.EphemeralAddon;
 import com.dfsek.terra.api.addon.BaseAddon;
 import com.dfsek.terra.lifecycle.LifecyclePlatform;
 
 
 public class QuiltPlatform extends LifecyclePlatform {
-    private static final Logger LOGGER = LoggerFactory.getLogger(QuiltPlatform.class);
     
     @Override
     protected Collection<BaseAddon> getPlatformMods() {
-        return QuiltLoader.getAllMods().stream().flatMap(mod -> {
-            String id = mod.metadata().id();
-            if(id.equals("terra") || id.equals("minecraft") || id.equals("java")) return Stream.empty();
-            try {
-                Version version = Versions.parseVersion(mod.metadata().version().raw());
-                return Stream.of(new EphemeralAddon(version, "quilt:" + id));
-            } catch(ParseException e) {
-                LOGGER.warn(
-                        "Mod {}, version {} does not follow semantic versioning specification, Terra addons will be unable to depend on " +
-                        "it.",
-                        id, mod.metadata().version().raw());
-            }
-            return Stream.empty();
-        }).collect(Collectors.toList());
+        return QuiltLoader.getAllMods()
+                          .stream()
+                          .flatMap(mod -> parseModData(mod.metadata().id(), mod.metadata().version().raw(), "quilt"))
+                          .collect(
+                                  Collectors.toList());
```

Comment:
fix this formatting lol

Reply:
this is a disabled platform. it can be fixed if it ever get renabled